src/scales/quantitativeScale.ts

```diff
@@ -20,10 +21,89 @@ module Plottable {
      */
     constructor(scale: D3.Scale.QuantitativeScale) {
       super(scale);
+      this._paddingExceptions = new Utils.Map<any, D>();
+      this._includedValues = new Utils.Map<any, D>();
     }
 
     protected _getExtent(): D[] {
-      return this._domainer.computeDomain(this._getAllExtents(), this);
+      var extents = this._getAllExtents().filter((extent) => extent.length > 0);
+      var defaultExtent = this._defaultExtent();
+      if (extents.length === 0) {
+        return defaultExtent;
+      }
+      var combinedExtent = [
+        Utils.Methods.min<D[], D>(extents, (extent) => extent[0], defaultExtent[0]),
+        Utils.Methods.max<D[], D>(extents, (extent) => extent[1], defaultExtent[1])
```

Comment:
If I don't add the generic, `min()` and `max()` will return `<any>`s (actually, it's possible that the line won't compile at all).